arch/arm/boot/dts/overlays/README

```diff
@@ -4988,6 +4988,12 @@ Params: clock-frequency         Display clock frequency (Hz)
                                 Set the default brightness. Normal range 1-16.
                                 (default 16).
         rotate                  Display rotation {0,90,180,270} (default 0)
+        rgb-order               Allow override of RGB order from DPI.
+                                0 = obey MEDIA_BUS_FMT (default)
+                                1 = BGR
+                                2 = GRB
+                                3 = BRG
+
```

Comment:
There are 6 permutations including GBR, RBG; I guess those are never wanted?

Reply:
Those permutations are not available on vc4.

https://github.com/raspberrypi/linux/blob/rpi-6.6.y/drivers/gpu/drm/vc4/vc4_dpi.c#L33-L41

Reply:
Ah, fair enough!

Meanwhile I just remembered RP1 would have a problem with them for "mode 2" (which really only works when the in-memory fb format is exactly RGB565) but that can't be helped.